data.yml

```diff
@@ -657,6 +657,72 @@ paths:
                 $ref: '#/components/schemas/StatusServiceUnavailableError'
       security:
         - services_token: []
+  /users/{userId}/connections/{connectionId}/purge:
+    post:
+      tags:
+        - Connections
+      summary: Purge connection data
+      description: >-
+        Use this to remove all user data for specific institution.
```

Comment:
Use this to remove all user data for a specific institution.

Reply:
@adam-atawneh can we add a section similar to the delete user end point with a caution statement?

https://api.basiq.io/reference/deleteuser

Reply:
@adam-atawneh updated